src/ert/config/queue_config.py

```diff
@@ -108,10 +108,11 @@ def create_local_copy(self) -> QueueConfig:
 
     @property
     def max_running(self) -> int:
+        max_running = 0
         for key, val in self.queue_options.get(self.queue_system, []):
             if key == "MAX_RUNNING":
-                return int(val)
-        return 0
+                max_running = int(val)
+        return max_running
```

Comment:
I think `return 0` was correct as this implies all realizations and we should have (in Scheduler) 
`if max_running==0 max_running=len(self._jobs)`